AmericanMedicalAssociation/ama-style-guide-2
# ADF-126: Prototype autocomplete dropdown

**Jira Ticket**
- [ADF-126: Prototype autocomplete dropdown](https://palantir.atlassian.net/browse/ADF-126)

## Description

- Add and style autocomplete states for the location search field.

## To Test
- Checkout the **ADF-126-prototype-autocomplete-dropdown** branch and cd into style guide, run gulp serve to spin up the pattern lab style guide.

- Visit **http://localhost:3000/?p=pages-dr-finder**  and verify when you type two letter in the location search field that the search dropdown list looks like the ticket modal.


## Relevant Screenshots/GIFs
<img width="1243" alt="Screen Shot 2022-10-12 at 2 31 54 PM" src="https://user-images.githubusercontent.com/54925022/195442420-74dcf923-f10b-4514-9442-5e974d695e02.png">
<img width="1224" alt="Screen Shot 2022-10-12 at 2 32 18 PM" src="https://user-images.githubusercontent.com/54925022/195442430-a6f62134-aa12-4f76-905c-92385c03c07a.png">
